README.md

```diff
@@ -61,57 +49,70 @@ This is a basic example on how to use the library. Based on a query the language
 ```python
 # imports
 import torch
-from transformers import GPT2Tokenizer
-from trl.gpt2 import GPT2HeadWithValueModel, respond_to_batch
-from trl.ppo import PPOTrainer
+from transformers import AutoTokenizer
+from trl import PPOTrainer, PPOConfig, AutoModelForCausalLMWithValueHead
+from trl.core import respond_to_batch
 
 # get models
-gpt2_model = GPT2HeadWithValueModel.from_pretrained('gpt2')
-gpt2_model_ref = GPT2HeadWithValueModel.from_pretrained('gpt2')
-gpt2_tokenizer = GPT2Tokenizer.from_pretrained('gpt2')
+gpt2_model = AutoModelForCausalLMWithValueHead.from_pretrained('gpt2')
+gpt2_model_ref = AutoModelForCausalLMWithValueHead.from_pretrained('gpt2')
```

Comment:
we can now use:

```suggestion
gpt2_model_ref = create_refrence_model(gpt2_model)
```

or we can not provide a reference model at all in which case the trainer will call above functions.